pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -91,13 +93,17 @@ let
             else "/bin";
 
   # 78 ESR won't build with rustc 1.47
-  inherit (if lib.versionAtLeast ffversion "82" then rustPackages else rustPackages_1_45)
+  inherit (if lib.versionAtLeast ffversion "82" then rustPackages_1_51 else rustPackages_1_45)
```

Comment:
Please add a comment how to discover the rust version Mozilla uses for their stable firefox builds.